package/com.unity.formats.usd/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Disabled plugins on unsupported platforms.
 - Fixed a bug causing PointInstances to be duplicated.
 - Fixed a bug causing an ApplicationException "all sample times in times and baseTime must either all be numeric or all be default" to be thrown when refreshing a PointInstancer.
+- Optimized triangulation to reduce the chance of a "Timed out while waiting for thread" error on importing a complex mesh.
```

Comment:
I did a comparison between the imports pre and post this change, and with my sample file, there was a reduction from 10 seconds to 5 seconds, which seems like a pretty big cut down in time

My test file has:
Batches: 42
Tris: 232.8M
Verts: 116.4M